common/app/model/content.scala

```diff
@@ -376,7 +377,8 @@ object Article {
       iosType = Some("Article"),
       javascriptConfigOverrides = javascriptConfig,
       opengraphPropertiesOverrides = opengraphProperties,
-      twitterPropertiesOverrides = twitterProperties
+      twitterPropertiesOverrides = twitterProperties,
+      isMinute = content.tags.isUSMinuteSeries
```

Comment:
Thoughts @desbo - shall we make the tag isMinute and add a new identifier (not sure where) for the specific campaign minute type (so we can add usMinute classes?). We mentioned refactoring this later on, but maybe we should update it now?